packages/react-components/src/TextAreaWithDropdown.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017-2020 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useCallback } from 'react';
+import { TextArea } from '@polkadot/react-components/TextArea';
+
+interface Props {
+  children?: React.ReactNode;
+  isError?: boolean;
+  isFocused?: boolean;
+  isReadOnly?: boolean;
+  rowsCount?: number;
+  onChange?: (value: string) => void;
+  value?: string;
+  id: string;
+}
+
+export default function TextAreaWithDropdown ({ children, id, isError, isFocused, isReadOnly, onChange, rowsCount, value }: Props): React.ReactElement<Props> {
+  const _onChange = useCallback(
+    ({ target: { value } }: React.ChangeEvent<HTMLTextAreaElement>): void => {
+      onChange && onChange(value);
+    },
+    [onChange]
+  );
+
+  return (
+    <>
```

Comment:
`<div>` so we can expand, add Labelled and classes for other component adjustments.